protocol/protocol.proto

```diff
@@ -190,6 +191,15 @@ message MemberPayload {
         bytes token_id = 3;
     }
 
+    message Pin { 
+        bytes event_id = 1;
```

Comment:
could you please add the userId of the creator of the pin here too?

Reply:
the creator address is in every single stream event already `bytes creator_address = 1;` it's verified against the signature of the event. Additionally, if we put it here users would be able to forge pins from other people.

Reply:
I see how we lost it, i'll fix the streamStateView_Members, one sec.